client/my-sites/stats/activity-log-date/index.jsx

```diff
@@ -0,0 +1,98 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import FoldableCard from 'components/foldable-card';
+import Button from 'components/button';
+import ActivityLogItem from '../activity-log-item';
+
+class ActivityLogDate extends Component {
```

Comment:
I had the same impression. Something that clearly indicates it's a _group_ of log items would be helpful.